mlir/utils/jenkins/Jenkinsfile

```diff
@@ -559,5 +560,51 @@ pipeline {
                 }
              }
         }
+        stage("Store a Release Build") {
+            when {
+                beforeAgent true;
+                equals expected: false, actual: params.weekly;
+                equals expected: false, actual: params.nightly;
+                not {equals expected: '', actual: params.releaseBranch}
+            }
+            agent {
+                docker {
+                    image dockerImage()
+                    args dockerArgs()
+                    label "${params.canXdlops ? 'gfx908' : 'rocm' }"
+                    alwaysPull true
+                }
+            }
+            stages{
+                stage ('Build and Archive a Release Branch') {
+                    steps {
+                        showEnv()
+                        sh "git checkout ${params.releaseBranch}"
```

Comment:
There might be a way to do this with Jenkins - maybe in combination with skipDefaultCheckout?

Reply:
It sounds doable. I'll give it a try too.